### PR TITLE
[ISSUE #39] 支持将Service的Labels同步至北极星Service以及Pod的Labels同步至北极星Instance

### LINKED ISSUE #39: 支持将Service的Labels、Pod的labels同步为service的metadata以及instance的metadata

### PR DESCRIPTION
**Pull request by chuntaojun**
**Please provide issue(s) of this PR:**
Fixes #39 

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Comment by zhangzerui20** [comment]
review

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -505,6 +506,7 @@ func (p *PolarisController) onEndpointUpdate(old, cur interface{}) {
 func (p *PolarisController) isPolarisEndpoints(endpoint *v1.Endpoints) (bool, string, error) {
 
 	// 先检查 endpoints 的 service 上是否有注解
+	p.podLister.List(labels.NewSelector())
```
[the remark this answers is not in the document]
Comment (chuntaojun) [reply pkg/controller/controller.go:509]: 估计是遗漏了，我删除下

File: pkg/util/address/address.go [hunk 2]
```diff
@@ -79,14 +91,22 @@ func GetAddressMapFromEndpoints(service *v1.Service, endpoint *v1.Endpoints,
...
+					Port:     int(port.Port),
+					PodName:  readyAds.TargetRef.Name,
+					Index:    tmpReadyIndex,
+					Weight:   tmpWeight,
+					Healthy:  true,
+					Protocol: port.Name,
+				}
+
+				if ok {
+					address.Metadata = pod.Labels
```
[the remark this answers is not in the document]
Comment (chuntaojun) [reply pkg/util/address/address.go:106]: 存量是指 ？这个应该是不会有存量兼容的问题吧应该？

File: pkg/controller/apis.go [hunk 3]
```diff
@@ -71,21 +71,33 @@ func (p *PolarisController) addInstances(service *v1.Service, address []address.
...
 
-	metadata := formMetadataFromService(service, p.config.PolarisController.ClusterName)
+	serviceMetadata := formMetadataFromService(service, p.config.PolarisController.ClusterName)
 
 	var instances []polarisapi.Instance
 
 	// 装载Instances
-	for _, i := range address {
-		*healthy = *healthy && i.Healthy
+	for i := range address {
```
[the remark this answers is not in the document]
Comment (chuntaojun) [reply pkg/controller/apis.go:79]: 哦，对，我处理下

File: pkg/util/address/address.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (chuntaojun) [reply pkg/util/address/address.go:106]: 这里应该不影响，因为最后会在 apis.go 里面处理的

**Review by zhangzerui20 — APPROVED** [review]

**Pull request merged**